docs/Contributing/Testing-and-local-development.md

```diff
@@ -448,3 +449,123 @@ MinIO also offers a web interface at http://localhost:9001. Credentials are `min
 You can configure the server to record and report trace data using OpenTelemetry or Elastic APM and use a tracing system like [Jaeger](https://www.jaegertracing.io/) to consume this data and inspect the traces locally.
 
 Please refer to [tools/telemetry](../../tools/telemetry/README.md) for instructions.
+
+## MDM setup and testing
+
+To run your local server with the MDM features enabled, you need to get certificates and keys.
+
+- [ABM setup](#abm-setup)
+- [APNs setup](#apns-setup)
+- [SCEP setup](#scep-setup)
+- [Running the server](#running-the-server)
+- [Testing MDM](#testing-mdm)
+
+### ABM setup
+
+To enable the [DEP](https://github.com/fleetdm/fleet/blob/main/tools/mdm/apple/glossary-and-protocols.md#dep-device-enrollment-program) enrollment flow, the Fleet server needs three things:
+
+1. A private key.
+1. A certificate.
+1. An encrypted token generated by Apple.
+
+### Private key + certificate
```

Comment:
Nit: this header and Encrypted token below belong under ABM setup, so I think they should have one more `#`.

Reply:
great catch, thank you